src/Components/LocationForm.js

```diff
@@ -24,9 +24,28 @@ export default function LocationForm({ onAddLocations }) {
     const fieldName = event.target.name;
     let fieldValue = event.target.value;
 
+    if (fieldName === 'address' && fieldValue.length >= 5) {
+      lookupLocation(fieldValue);
+    }
+
     setLocation({ ...location, [fieldName]: fieldValue });
   }
 
+  function lookupLocation(address) {
+    const apiUrl = `https://nominatim.openstreetmap.org/search?q=${address}&format=json`; // URLencode
+    fetch(apiUrl)
+      .then((result) => result.json())
+      .then((data) => {
+        // Rosenheimer 143, München
```

Comment:
If you no longer need this comment, do not forget to delete it. :)

Reply:
Alright, thank you :)